frontend/src/components/bucket/BucketsTable.vue

```diff
@@ -3,7 +3,7 @@ import { storeToRefs } from 'pinia';
 import DataTable from 'primevue/datatable';
 import Column from 'primevue/column';
 import Button from 'primevue/button';
-
+import { RouteNames } from '@/utils/constants';
 import { useBucketStore } from '@/store/bucketStore';
```

Comment:
Just as an optional style note - when we're importing from our own code, we've historically alphabetized based on the from path; so in this case this line should be below the useBucketStore as store comes before utils. Don't need to follow this pattern everywhere but I figure it's not a bad thing to continue doing.
For the primevue imports, we should alphabetize this if possible too.